### PR TITLE
Fixed build, fixed run, and improvements to usage

### PR DESCRIPTION
**Pull request by onionhammer**
- No hardcoded endpoint
- Added missing packages (solution broken otherwise)
- console.log instead of alert
- <remove> static content before <mimeMap>

### CODE REVIEW

**Comment by parakhj** [comment]
@omer-iqbal could you look at the changes?

File: UserJourneyRecorder/UserJourneyRecorderWebApp/UserJourneyRecorderWebApp.csproj [hunk 1]
```diff
@@ -74,6 +84,7 @@
     <Content Include="packages.config" />
     <Content Include="languages\en-us_102.json" />
     <None Include="Properties\PublishProfiles\B2CRecorder.pubxml" />
+    <None Include="Properties\PublishProfiles\CumminsUserJourney - Web Deploy.pubxml" />
```
Comment (gsacavdm) [inline UserJourneyRecorder/UserJourneyRecorderWebApp/UserJourneyRecorderWebApp.csproj]: Is this intended? The file doesn't seem generic enough and isn't included in the PR.

Comment (onionhammer) [reply UserJourneyRecorder/UserJourneyRecorderWebApp/UserJourneyRecorderWebApp.csproj]: Nope. My bad! I'll remove it when I get a chance

Comment (onionhammer) [reply UserJourneyRecorder/UserJourneyRecorderWebApp/UserJourneyRecorderWebApp.csproj]: Removed

**Comment by onionhammer** [comment]
Bump - whats the status on this PR?

**Comment by monteledwards** [comment]
Much needed - does not build without changes. Thanks for taking the time to make PR @onionhammer

**Pull request merged**